Scripts/SDL/Kveer.sdl

```diff
@@ -59,3 +59,16 @@ VAR BOOL    kverMystLinkingBookVis[1]    DEFAULT=0 DEFAULTOPTION=VAULT
 ## Reward Clothing
 
 }
+
+STATEDESC Kveer
+{
+	VERSION 2
+
+# Boolean variables
+VAR BOOL    kverMystLinkingBookVis[1]    DEFAULT=0 DEFAULTOPTION=VAULT
+
+## Reward Clothing
+VAR BOOL    kverAtrusVis[1]              DEFAULT=0 DEFAULTOPTION=VAULT
+VAR BOOL    kverAtrusGet[1]              DEFAULT=1
```

Comment:
I don't think we want this variable (kverAtrusGet); all possible variations of K'veer may be shared with multiple players, so we don't want the clothing item to disappear for future players when a single player collects it.

Reply:
I mentioned this in my supplemental comment above actually. That's actually why there's two states.

In normal cases, it appears the get variable and the show/hide variable use the same SDL bool, so when you click the item to get it, it also toggles the show/hide, causing the item to disappear...which is annoying if someone grabs your clothing item before you do. An exception to this is Catherine's tunic on Myst Island...it only has the get script and no show/hide script, so it doesn't disappear when you click it.

Having the two separate bool variables will allow us to appear/disappear the clothing item separately while still allowing the xTakableClothing script to remain active.

Reply:
Does xTakableClothing not allow for the SDL variable to be empty?

Reply:
> 
> 
> Does xTakableClothing not allow for the SDL variable to be empty?

I...haven't tried that. I'll check it out.

At the same time, we'd probably still want the show/hide bool so we can have it "magically" appear sometime after the big update itself, yes? Or would we rather it just be constantly on? Honestly, I'm fine with either way. 😄

Reply:
Yeah, I think keeping the global show/hide is definitely a good thing... If we really want to change the SDL 👅